xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3107,6 +3107,9 @@ xe.templateprovider.action=Action on create
 xe.templateprovider.action.info=The action to execute when the create button is pushed, you can configure here whether the new page is saved before it is opened for edition or not.
 xe.templateprovider.terminal=Terminal Page
 xe.templateprovider.terminal.hint=Whether or not to create terminal documents by default when using this template provider.
+xe.templateprovider.description=Description
+xe.templateprovider.icon=Icon
+xe.templateprovider.icon.info=See the {0}full list of icons{1}.
```

Comment:
Yes it is, especially since we're at RC1 now for 8.2... :)

Now we need to start being more carefully to stop adding technical debt. Hopefully we have some tools to migrate the translations.